packages/wonder-blocks-core/src/hooks/use-force-update.ts

```diff
@@ -13,25 +13,16 @@ import * as React from "react";
  * @returns {() => void} A function that forces the component to update.
  */
 export const useForceUpdate = (): (() => void) => {
-    const updatePendingRef = React.useRef(false);
-    const [, setUpdateToggle] = React.useState(false);
+    const [, setUpdateState] = React.useState({});
 
     const forceUpdate = React.useCallback(() => {
-        if (updatePendingRef.current) {
-            // If an update is already pending, then we do nothing.
-            return;
-        }
-
-        // Otherwise, if we haven't been asked to force an update since our
-        // last render then we toggle the state to invoke a render.
-        setUpdateToggle((toggle) => !toggle);
-        updatePendingRef.current = true;
+        // We leverage here that every new object instance will be seen
+        // as a state change. This is a little hacky but it works better than
+        // a boolean that would just flip-flop and could not trigger a render,
+        // or a random number that could repeat values and also then not
+        // trigger a render. This will always work.
+        setUpdateState({});
```

Comment:
Neat!  This feels like a variation on my "random number" suggestion.

Reply:
Yes, I tried that but abandoned it for two reasons:

1. It didn't guarantee a fix to the original issue since a random number could repeat a number already seen.
2. It was more complicated.

Reply:
I like that this guarantees the state value is always new.  Good call. 🙂